systori/apps/equipment/api.py

```diff
@@ -15,7 +17,11 @@ class EquipmentModelViewSet(viewsets.ModelViewSet):
     queryset = Equipment.objects.prefetch_related("equipment_type").all()
     serializer_class = EquipmentSerializer
     permission_classes = (HasLaborerAccess,)
+    refuelingstops_response = OpenApiResponse(
+        "Refueling Stops", RefuelingStopSerializer(many=True)
+    )
 
+    @swagger_auto_schema(responses={200: refuelingstops_response})
     @action(detail=True)
     def refuelingstops(self, request, pk=None):
```

Comment:
Currently, this method returns all of the `RefuelingStop`s, as a result the output json contains an array of `RefuelingStop`s. Is there a way to limit or paginate this response? Like how most of the `list` options work. Since this is in fact a `list` operation.